docusaurus/docs/reference/ha/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_label: Overview
+sidebar_position: 05
+---
+
+# Controller HA 
+
+## Overview
+
+Ziti controllers can be run in a cluster for high availablity and performance scaling.
+
+:::warning
+
+**NOTE: Controller HA is still in Beta** 
+
+It's quite functional now, but we are continuing to test and refine before we mark it GA.
+:::
+
+
+### For SDK Clients/Tunnelers
```

Comment:
Turns out a lot of the 'what' is here under this heading. I'd say just remove this heading and it reads a bit better. the lead sentence needs a bit more gusto though imo. Just a bit of prose before jumping into benefits...

Also benefits/limitations are reasonable sub-headings here imo